Numerge.Gui/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Logging.Serilog;
+using Avalonia.ReactiveUI;
+using Numerge.Gui.ViewModels;
+using Numerge.Gui.Views;
+
+namespace Numerge.Gui
+{
+    class Program
+    {
+        // Initialization code. Don't use any Avalonia, third-party APIs or any
+        // SynchronizationContext-reliant code before AppMain is called: things aren't initialized
+        // yet and stuff might break.
+        public static void Main(string[] args) => BuildAvaloniaApp().Start(AppMain, args);
+
+        // Avalonia configuration, don't remove; also used by visual designer.
+        public static AppBuilder BuildAvaloniaApp()
+            => AppBuilder.Configure<App>()
+                .UsePlatformDetect()
+                .LogToDebug()
+                .UseReactiveUI();
+
+        // Your application's entry point. Here you can initialize your MVVM framework, DI
+        // container, etc.
+        private static void AppMain(Application app, string[] args)
```

Comment:
Probably worth using the new [Avalonia lifetimes](https://github.com/AvaloniaUI/Avalonia/wiki/Application-lifetimes) here? And moving initialization logic to the `App.OnFrameworkInitializationCompleted` method, see [a sample app](https://github.com/reactiveui/ReactiveUI.Samples/blob/master/avalonia/ReactiveUI.Samples.Suspension/App.xaml.cs#L22) for the guidance.